File: mobile/modules/bluetooth-sdk/android/src/main/java/com/veiller/bluetoothsdk/VeillerBluetoothSdkDebug.kt
package com.veiller.bluetoothsdk

object VeillerBluetoothSdkDebug {
    @JvmStatic
    fun setOtaVersionUrl(
        sdk: VeillerBluetoothSdk,
        otaVersionUrl: String,
    ) {
        sdk.setOtaVersionUrl(otaVersionUrl)
    }

    @JvmStatic
    fun getOtaVersionUrl(sdk: VeillerBluetoothSdk): String = sdk.getOtaVersionUrl()
}

File: mobile/modules/bluetooth-sdk/ios/Source/VeillerBluetoothSDKDebug.swift
import Foundation

@MainActor
public enum VeillerBluetoothSDKDebug {
    public static func setOtaVersionUrl(_ otaVersionUrl: String, on sdk: VeillerBluetoothSDK) throws {
        try sdk.setOtaVersionUrl(otaVersionUrl)
    }

    public static func getOtaVersionUrl(on sdk: VeillerBluetoothSDK) throws -> String {
        try sdk.getOtaVersionUrl()
    }
}
